src/mscorlib/src/Internal/Runtime/Augments/EnvironmentAugments.cs

```diff
@@ -17,5 +18,11 @@ public static class EnvironmentAugments
         public static bool HasShutdownStarted => Environment.HasShutdownStarted;
         public static string StackTrace => Environment.StackTrace;
         public static int TickCount => Environment.TickCount;
+        public static string GetEnvironmentVariable(string variable) => Environment.GetEnvironmentVariableCore(variable);
+        public static string GetEnvironmentVariable(string variable, EnvironmentVariableTarget target) => Environment.GetEnvironmentVariableCore(variable, target);
```

Comment:
See my other comment above- I thought you didn't want me to move the registry code down.  https://github.com/dotnet/corefx/issues/13931